indi-atik/atik_ccd.cpp

```diff
@@ -222,8 +237,9 @@ bool ATIKCCD::initProperties()
 
     // Version information
     IUFillText(&VersionInfoS[VERSION_API], "VERSION_API", "API", std::to_string(ArtemisAPIVersion()).c_str());
+    IUFillText(&VersionInfoS[VERSION_DLL], "VERSION_DLL", "DLL", std::to_string(ArtemisDLLVersion()).c_str());
```

Comment:
What is rationale behind introducing VERSION_DLL? It does not apply to linux, arm and mac libs.

Reply:
Well, I'd agree if there was a document stating so. You had this information in the past? I couldn't find it in the SDK docs.

Reply:
So it returns VERSION_DLL = 1.4.0.5 for linux, arm and mac libs? It could be the case that it actually returns windows lib (dll) version instead. I do not recall anything like version 1.4.0.5 in the past.

Reply:
I'll keep 1.4.0.5 as version because that matches the SDK versions properly, see investigation in issue #105.
I have no information on what to use between API and DLL, so I can remove my change here, OK.

Reply:
See my comment above to libatik/CMakeLists.txt